src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1438,6 +1443,33 @@ public static bool IsInstanceContext(this SyntaxTree syntaxTree, int position, S
             return false;
         }
 
+        private static ArrowExpressionClauseSyntax GetExpressionBodyFromMember(MemberDeclarationSyntax containingMember)
+        {
+            if (containingMember != null)
+            {
+                switch (containingMember.Kind())
+                {
+                    case SyntaxKind.PropertyDeclaration:
+                        return ((PropertyDeclarationSyntax)containingMember).ExpressionBody;
+                    case SyntaxKind.MethodDeclaration:
+                        return ((MethodDeclarationSyntax)containingMember).ExpressionBody;
+                    case SyntaxKind.IndexerDeclaration:
+                        return ((IndexerDeclarationSyntax)containingMember).ExpressionBody;
+                    default:
```

Comment:
No, I think the proposal would be to move it to something like SyntaxNodeExtensions or SyntaxFacts and have it no care about the instance/static distinction.  It's generally useful to be able to get to the body of an expression bodied member regardless of member type.